static/js/settings_user_groups.js

```diff
@@ -303,6 +304,75 @@ export function set_up() {
         });
     });
 
+    $("#user-groups").on("click", ".edit", function () {
+        const group_id = Number.parseInt($(this).parents(".user-group").attr("id"), 10);
+        if (!can_edit(group_id)) {
+            return;
```

Comment:
We should instead hide the button if user is not allowed to edit a user group.

Reply:
https://user-images.githubusercontent.com/85362194/158947505-047bf5c6-02be-4ab9-8dff-a8b8492f9089.mov

Here i have hidden the buttons container when the user is not allowed to edit a group and used ```dialog_widget``` instead of ```confirm_widget```